tests/support.rs

```diff
@@ -0,0 +1,34 @@
+use std::{
+    fs::{create_dir, OpenOptions},
+    io::Write,
+    path::PathBuf,
+};
+
+use vector::test_util::{temp_dir, temp_file};
+
+/// Creates a file with given content
+pub fn create_file(config: &str) -> PathBuf {
+    let path = temp_file();
+    overwrite_file(path.clone(), config);
+    path
+}
+
+/// Overwrites file with given content
+pub fn overwrite_file(path: PathBuf, config: &str) {
+    let mut file = OpenOptions::new()
+        .create(true)
+        .write(true)
+        .truncate(true)
+        .open(path)
+        .unwrap();
```

Comment:
Does this need to protect against write/read races by creating a temporary and renaming over the original?

Reply:
Oh, nevermind, I see this is just moved code.